dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.facesProvider-3.0.0.MyFaces.feature

```diff
@@ -0,0 +1,9 @@
+# This private feature corresponds to using a JSF-providing feature
```

Comment:
Given that this is a renamed feature io.openliberty and uses faces instead of jsf we need to make sure this and the other jsf provider features can't be loaded at the same time.  Have we done any testing? Perhaps a discussion in the jakarta channel is necessary as there are likely other providers that have hit the same issue.

Reply:
I tried with faces-3.0 and jsfContainer-2.3 and got the following: 

```
[ERROR   ] CWWKF0044E: The features servlet-4.0 and el-4.0 cannot be loaded at the same time.  The feature servlet-4.0 of Java EE 8 is incompatible with the feature el-4.0 of Jakarta EE 9.  The configured features jsfContainer-2.3 and faces-3.0 include an incompatible combination of features.  Your configuration is not supported.  Update the configuration to use features that support either the Jakarta EE or Java EE programming models, but not both.
[ERROR   ] CWWKE0702E: Could not resolve module: .....
[AUDIT   ] CWWKF0012I: The server installed the following features: [cdi-2.0, el-3.0, faces-3.0, jndi-1.0, jsfContainer-2.3, jsp-2.3, jsp-3.0, websocket-1.1].
```

Reply:
With  #13936 resolved, I tried with faces-3.0 and jsfContainer-2.3 and got the following:
`
[ERROR   ] CWWKF0044E: The features jsfContainer-2.3 and faces-3.0 cannot be loaded at the same time.  The feature jsfContainer-2.3 of Java EE 8 is incompatible with the feature faces-3.0 of Jakarta EE 9.  The configured features jsfContainer-2.3 and faces-3.0 include an incompatible combination of features.  Your configuration is not supported.  Update the configuration to use features that support either the Jakarta EE or Java EE programming models, but not both.`

Looks good :)